server/jobs/job_manager.go

```diff
@@ -16,26 +16,26 @@ import (
 )
 
 type JobManager struct {
+	env            *mscalendar.Env
+	papi           cluster.JobPluginAPI
 	registeredJobs sync.Map
 	activeJobs     sync.Map
-	env            mscalendar.Env
```

Comment:
Why was `env` changed to a pointer?

Reply:
The previous `golangci-lint` configuration was giving errors here but now it's not. Do you want me to revert this change?

Reply:
Yes, please

Reply:
Done.